PIL/__init__.py

```diff
@@ -11,8 +11,10 @@
 
 # ;-)
 
-VERSION = '1.1.7'  # PIL version
```

Comment:
So we're ditching the old PIL version.

The build fails because PIL/Image.py still tries to import `VERSION`, but doesn't look like Image.py or anything that import it actually needs it.